src/components/form-editor/form-editor.scss

```diff
@@ -107,3 +107,16 @@ button {
   display: flex;
   align-items: center;
 }
+
+.dropdownContainer {
+  display: flex;
+  align-items: center;
+  :global(.cds--label) {
+    margin-bottom: 0 !important;
+  }
+}
+
+.dropdownLabel {
+  margin-right: 0.25rem;
+  white-space: nowrap;
+}
```

Comment:
```suggestion
```

This is no longer in use, right?

Reply:
Yep, cleaned up the styles, and added a bit to reduce the gap between the label and the dropdown